subsys/net/openthread/rpc/client/ot_rpc_netdiag.c

```diff
@@ -251,14 +251,10 @@ otError otThreadSendDiagnosticGet(otInstance *aInstance, const otIp6Address *aDe
 	receive_diag_get_cb = aCallback;
 	receive_diag_get_cb_context = aCallbackContext;
 
-	NRF_RPC_CBOR_ALLOC(&ot_group, ctx, 1 + OT_IP6_ADDRESS_SIZE + 4 + aCount * 2);
+	NRF_RPC_CBOR_ALLOC(&ot_group, ctx, 1 + OT_IP6_ADDRESS_SIZE + 4 + aCount);
 	nrf_rpc_encode_buffer(&ctx, aDestination->mFields.m8, OT_IP6_ADDRESS_SIZE);
 	nrf_rpc_encode_uint(&ctx, aCount);
```

Comment:
Now it is even more redundant to encode the count as a separate int :). Could you simplify this as well?

Reply:
oh yeah, right... Thanks, updated the code :)